src/node.rs

```diff
@@ -208,17 +250,20 @@ impl Sprawl {
         Ok(&self.forest.nodes[id].layout)
     }
 
+    /// Indicates that the layout of this node and its children must be recomputed
     pub fn mark_dirty(&mut self, node: Node) -> Result<(), Error> {
         let id = self.find_node(node)?;
         self.forest.mark_dirty(id);
         Ok(())
     }
 
+    /// Does the layout of this node (and its children) need to be recomputed
```

Comment:
This is a question, but without the question mark which is kind of confusing. Maybe something like this?

```suggestion
    /// Indicates whether the layout of this node (and its children) need to be recomputed
```